app/models/rental.py

```diff
@@ -0,0 +1,10 @@
+from flask import current_app
+from app import db
+# from app.models.video import Video
+# from app.models.customer import Customer
+
+class Rental(db.Model):
```

Comment:
Note:  It would make a great deal of sense to add more methods to the model classes to handle the "business logic" of the application.  Things like checking out a video and checking in a video.

This would accomplish a few things:

1.  It would separate the specific application logic (into the models) away from the part getting the requests and sending back the responses (routes.py).
1.  It would lead to dryer code so your routes functions are not too long.
1.  It would make the code more reusable by decouping the specific API logic from the video store POS (Point of Service) logic.